### ISSUE TITLE
Corrupted or incomplete model files: model/SGF_model and model/SIF_model

### ISSUE THREAD

**Issue by bearcfaria13** [FrankWanger/ML_Peptide#4]
Hi, thanks for sharing this project.

I noticed that the files inside model folder appears to be incomplete. Their size is only about 8 KB, and when opened it contains only the header of a scikit‑learn pickle (e.g., references to KNeighborsClassifier) without the full binary content.
This causes **errors when loading the model in the notebook.**

It looks like the file **may have been truncated during upload**.
Could you please check and upload the full model file?

Thanks again for your work on this project.

**Comment by FrankWanger** [comment]
Hi Beatriz,

Thanks for your interest in this project. I have just tested the code and prediction on my end and it seemed okay. Do you mind providing me your environment info (including python/rdkit/scikit version etc.?) and also the exact error you've encountered?

The following is the output of my running prediction:

> 100%|██████████| 1/1 [00:00<00:00, 13.43it/s]
Prediting SIF Stability...
Prediting SGF Stability...
Predicted SIF/SGF stability saved to the original file:  Sample_Sequence.csv
/opt/anaconda3/envs/peptide/lib/python3.8/site-packages/sklearn/neighbors/_classification.py:211: FutureWarning: Unlike other reduction functions (e.g. `skew`, `kurtosis`), the default behavior of `mode` typically preserves the axis it acts along. In SciPy 1.11.0, this behavior will change: the default value of `keepdims` will become False, the `axis` over which the statistic is taken will be eliminated, and the value None will no longer be accepted. Set `keepdims` to True or False to avoid this warning.
  mode, _ = stats.mode(_y[neigh_ind, k], axis=1)
Compound	SMILES	Stability_in_SIF	Stability_in_SGF
0	Oxytocin	CC[C@H](C)[C@H]1C(=O)N[C@H](C(=O)N[C@H](C(=O)N...	Not Stable	Stable

For your reference my env: osx-arm64/ python 3.8.20/ rdkit (2022.09.5)/ scikit 0.24.2, which was installed through `conda create -n peptide -c conda-forge python=3.8 scikit-learn=0.24.2 xgboost=1.3.3 pandas=1.3.0 numpy=1.20.3 tqdm` followed by `conda install conda-forge::rdkit`

**Comment by bearcfaria13** [comment]
Thank you for sharing your environment details! Everything is running smoothly now.
I must have missed a small detail during my initial setup.
